scripts/umbrel-os/external-storage/update-from-sdcard

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+UMBREL_ROOT="$(readlink -f $(dirname "${BASH_SOURCE[0]}")/../../..)"
+SD_MOUNT_POINT="/sd-root"
+SD_UMBREL_ROOT="${SD_MOUNT_POINT}${UMBREL_ROOT}"
+
+check_root () {
+  if [[ $UID != 0 ]]; then
+    echo "This script must be run as root"
+    exit 1
+  fi
+}
+
```

Comment:
Should we add a dependency check here for `jq`?

Reply:
Ah yes, good spot.